docs/reference/rest_api_communities.md

```diff
@@ -86,15 +86,15 @@ Content-Type: application/json
   "id": "399a3cdc-d2ba-4f63-8b3a-9c2c977a5dd3",
   "revision_id": 1,
   "links": {
-    "self": "{scheme+hostname}/api/communities/{community_id}",
+    "self": "{scheme+hostname}/api/communities/{id}",
```

Comment:
I have completely removed `comid`, `comm_id` and all similar instances. I thought we should just leave {id} since we're accepting both, unsure if it will be confusing or not.